apps/mobile/src/libs/components/checkbox-categories-form/checkbox-categories-form.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+	Button,
+	Checkbox,
+	LoaderWrapper,
+	Text,
+	View,
+} from "~/libs/components/components";
+import { BaseColor, DataStatus } from "~/libs/enums/enums";
+import {
+	useAppDispatch,
+	useAppForm,
+	useAppSelector,
+	useCallback,
+	useEffect,
+	useFormController,
+	useState,
+} from "~/libs/hooks/hooks";
+import { globalStyles } from "~/libs/styles/styles";
+import { categoriesSelectedValidationSchema } from "~/packages/categories/categories";
+import { actions as categoriesActions } from "~/slices/quiz/quiz";
+
+import {
+	CATEGORIES_FIELD_NAME,
+	CATEGORIES_FORM_DEFAULT_VALUES,
+	CHECK_ALL_CATEGORIES_NAME,
+	EMPTY_ARRAY_LENGTH,
+} from "./libs/constants/constants";
+import {
+	getAllCategoryIds,
+	getCategoriesSortedByScore,
+	getSelectedCategoryIds,
+} from "./libs/helpers/helpers";
+import { type SelectedCategoryIds } from "./libs/types/types";
+
+type Properties = {
+	onSubmit?: (payload: { categoryIds: SelectedCategoryIds }) => void;
```

Comment:
```suggestion
	onSubmit: (payload: { categoryIds: SelectedCategoryIds }) => void;
```
It should be a mandatory field